From 6c0ec2a675b299c0ba676d7179fc20dc49af09b3 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 13 Jul 2018 06:36:42 -0600
Subject: [PATCH 07/45] msm: ipa: Add declaration of ipa_del_hdr_by_user

An incomplete backport/patch failed to add this declaration to
ipa_i.h

Change-Id: I34d07ec7362c2aff8e652fec9bf30287aa51efb5
Fixes: c5b59371f93d ("msm: ipa: Prevent multiple header deletion from user space")
Signed-off-by: Kevin F. Haggerty <haggertk@lineageos.org>
---
 drivers/platform/msm/ipa/ipa_i.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/platform/msm/ipa/ipa_i.h b/drivers/platform/msm/ipa/ipa_i.h
index 0e6f7208ca2..deeffb286d7 100644
--- a/drivers/platform/msm/ipa/ipa_i.h
+++ b/drivers/platform/msm/ipa/ipa_i.h
@@ -827,6 +827,7 @@ void ipa_enable_clks(void);
 void ipa_disable_clks(void);
 void ipa_inc_client_enable_clks(void);
 void ipa_dec_client_disable_clks(void);
+int ipa_del_hdr_by_user(struct ipa_ioc_del_hdr *hdls, bool by_user);
 int __ipa_del_rt_rule(u32 rule_hdl);
 int __ipa_del_hdr(u32 hdr_hdl, bool by_user);
 int __ipa_release_hdr(u32 hdr_hdl);
-- 
2.17.1

